src/frontend/screens/MapScreen/GPSPermissions/GPSBackgroundPermissionDisabled.tsx

```diff
@@ -0,0 +1,83 @@
+import * as React from 'react';
+import {defineMessages, useIntl} from 'react-intl';
+import {StyleSheet, View, Linking} from 'react-native';
+import {Text} from '../../../sharedComponents/Text';
+import {MapPinErrorIconSmall} from '../../../sharedComponents/MapPinErrorIcon/MapPinErrorIconSmall';
+import {Button} from '../../../sharedComponents/Button';
+import * as Location from 'expo-location';
+
+const handleOpenSettings = () => {
+  Linking.sendIntent('android.settings.LOCATION_SOURCE_SETTINGS');
+};
+
+const m = defineMessages({
+  useLocation: {
+    id: 'screens.MapScreen.GPSBackgroundPermissionDisabled.useLocation',
+    defaultMessage: 'Use Your Location',
+  },
+  collectsLocation: {
+    id: 'screens.MapScreen.GPSBackgroundPermissionDisabled.collectsLocation',
+    defaultMessage:
+      'CoMapeo collects location data to track your route on the map, even when the app is closed. Your location data is only stored on your device by default, and is not stored or sent anywhere.',
+  },
+  turnOn: {
+    id: 'screens.MapScreen.GPSBackgroundPermissionDisabled.turnOn',
+    defaultMessage: 'Turn On',
+  },
+});
+
+type GPSBackgroundPermissionDisabledProp = {
+  setBackgroundStatusGranted: (status: boolean) => void;
+};
+
+export const GPSBackgroundPermissionDisabled = ({
+  setBackgroundStatusGranted,
+}: GPSBackgroundPermissionDisabledProp) => {
+  const {formatMessage} = useIntl();
+
+  async function askBackgroundLocationPermission() {
+    const backgroundPermission =
+      await Location.requestBackgroundPermissionsAsync();
+
+    if (backgroundPermission.granted) {
+      setBackgroundStatusGranted(true);
+      return;
+    }
+
+    if (backgroundPermission.canAskAgain) {
```

Comment:
based on the [docs](https://docs.expo.dev/versions/v50.0.0/sdk/location/#permissionresponse), i think this should be flipped. otherwise i commonly run into situations where pressing the button does nothing.

> Indicates if user can be asked again for specific permission. If not, one should be directed to the Settings app in order to enable/disable the permission.

```suggestion
    if (!backgroundPermission.canAskAgain) {
```

Reply:
https://github.com/digidem/comapeo-mobile/pull/810/commits/290ca88b575d19f0103839b117e82d6bb254d3c8

Reply:
seems to be pointing to the other file, not this one

Reply:
whoops! I had it on my vscode unsaved

Reply:
https://github.com/digidem/comapeo-mobile/pull/810/commits/11a34c8341d672175c178fe0eb8ea3af9e20e31e